docker-compose.yml

```diff
@@ -1,19 +1,62 @@
-version: '3'
+version: '2.1'
 services:
-  api:
-    image: openrecord/turntable:latest
+  vinyl:
+    build:
+      context: .
+      dockerfile: docker/vinyl/Dockerfile
     ports:
-      - 7001:7001
+      - 8080:8080
     environment:
-      PORT:     # Optionally specify a port for the API here.
-      DB_HOST: mysql
-    depends_on:
-      - mysql
-  mysql:
-    image: mysql:5.7
-    volumes:
-      - ./docker/mysql/mount:/var/lib/mysql # Add a mount for all mysql files, so you can persist across container tear-downs.
+      HOST: docker
+    networks: [main]
+
+  #TODO: wait on prisma healthcheck
+  turntable-deploy:
+    image: openrecord/turntable:prisma-cli
+    depends_on: [prisma]
+    command: npx prisma deploy
```

Comment:
Very uncomfortable with the fact that we deploy schema changes to production to run local tests.

Reply:
Yeah this was my hack to get a "local" turntable up and running. It's just deploying the code locally. Will be cleaning this up when we add the backend.